### PR TITLE
Add CLI to fail/complete activity

### PR DESCRIPTION
**Pull request by longquanzheng**
https://github.com/uber/cadence/issues/738

tested locally:
```
longer@~/gocode/src/github.com/uber/cadence:(act)$ ./cadence --do samples-domain wf act fail -w helloworld_47deca8f-8aae-4fe8-8481-460581e521cc -r 3ea946d0-00b9-45ed-9b42-2ba9e08b4b74 --aid 0 --reason "manual failing" --detail "{}" --identity "longer"
Fail activity successfully.
```

### CODE REVIEW

File: tools/cli/commands.go [hunk 1]
```diff
@@ -1081,6 +1081,68 @@ func ResetWorkflow(c *cli.Context) {
...
+}
+
+// FailActivity fails an activity
+func FailActivity(c *cli.Context) {
+	domain := getRequiredGlobalOption(c, FlagDomain)
+	wid := getRequiredOption(c, FlagWorkflowID)
+	rid := getRequiredOption(c, FlagRunID)
+	activityID := getRequiredOption(c, FlagActivityID)
+	if len(activityID) == 0 {
+		ErrorAndExit("wrong activityID", fmt.Errorf("activityID cannot be empty"))
```
Comment (vancexu) [inline tools/cli/commands.go]: ditto

File: tools/cli/workflow.go [hunk 2]
```diff
@@ -218,3 +224,71 @@ func newWorkflowCommands() []cli.Command {
...
+					Name:  FlagRunIDWithAlias,
+					Usage: "RunID",
+				},
+				cli.StringFlag{
+					Name:  FlagActivityIDWithAlias,
+					Usage: "The activityID to operate on",
+				},
+				cli.StringFlag{
+					Name:  FlagResult,
+					Usage: "result of the activity",
```
Comment (vancexu) [inline tools/cli/workflow.go]: Picky: Capitalized first letter.

File: tools/cli/workflow.go [hunk 3]
```diff
@@ -218,3 +224,71 @@ func newWorkflowCommands() []cli.Command {
...
+				},
+				cli.StringFlag{
+					Name:  FlagIdentity,
+					Usage: "identity of the activity result",
```
Comment (vancexu) [inline tools/cli/workflow.go]: This should be identity of operator (or worker)

Comment (longquanzheng) [reply tools/cli/workflow.go]: Makes sense

File: tools/cli/workflow.go (hunk 2 above)
Comment (longquanzheng) [reply tools/cli/workflow.go]: OK

File: tools/cli/commands.go (hunk 1 above)
Comment (longquanzheng) [reply tools/cli/commands.go]: OK

File: tools/cli/commands.go [hunk 4]
```diff
@@ -1081,6 +1081,68 @@ func ResetWorkflow(c *cli.Context) {
...
 }
 
+// CompleteActivity completes an activity
+func CompleteActivity(c *cli.Context) {
+	domain := getRequiredGlobalOption(c, FlagDomain)
+	wid := getRequiredOption(c, FlagWorkflowID)
+	rid := getRequiredOption(c, FlagRunID)
+	activityID := getRequiredOption(c, FlagActivityID)
+	if len(activityID) == 0 {
+		ErrorAndExit("wrong activityID", fmt.Errorf("activityID cannot be empty"))
```
[the remark this answers is not in the document]
Comment (longquanzheng) [reply tools/cli/commands.go]: OK

**Pull request merged**